.github/workflows/docs-release.yml

```
@@ -83,7 +83,7 @@ jobs:
         git branch
     - name: Install Python Dependencies
       run: |
-        pip install daal-devel impi-devel
+        pip install daal-devel impi-devel scikit-learn
         pip install -r dependencies-dev
         pip install -r requirements-doc.txt
```

```suggestion
        pip install daal-devel impi-devel
        pip install -r dependencies-dev
        pip install -r requirements-doc.txt
        pip install -r requirements-test.txt
```

to match the docs CI

---

I don't think this would be beneficial, because it pulls docstrings from sklearn (such as the metadata routing calls), so it's better for it to have always the latest.